.github/workflows/pamoauthrpmbuild.yml

```diff
@@ -9,14 +9,20 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
+
+    - name: Set up build directory
+      run: |
+        mkdir -p ${{github.workspace}}/build_output
+        sudo chown 1000:1000 ${{github.workspace}}/build_output
+
     - name: Build RPMS 
       run: |
           cd ./packaging/rpm
           docker build -t pamoauth2device-rpm-build .
-          docker run --rm -u root -v ${{github.workspace}}:/data pamoauth2device-rpm-build cp -r rpmbuild/RPMS /data
+          docker run --rm -u 1000:1000 --user builder -v ${{github.workspace}}/build_output:/data pamoauth2device-rpm-build:latest cp -r rpmbuild/RPMS /data
```

Comment:
If you look at the `Dockerfile` the default user is `builder` and it should by default be assigned 1000 for its UID and GID.